File: NetFramework/FlexReport/VB/ExternalDataSource/ExtRecordset/MyExtRecordsetForm.vb
Public Class MyExtRecordsetForm
    Public Function Edit(recordset As MyExtRecordset) As Boolean
        tbRecCount.Text = recordset.RecCount.ToString()
        Dim result As Boolean = ShowDialog() = System.Windows.Forms.DialogResult.OK
        If (result) Then
            recordset.RecCount = Integer.Parse(tbRecCount.Text)
        End If
        Return result
    End Function
End Class

File: ExcludedSamples/Reports/C1Report/Cs/ExternalDataSource/ExtRecordset/MyExtRecordsetForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ExtRecordset
{
    public partial class MyExtRecordsetForm : Form
    {
        public MyExtRecordsetForm()
        {
            InitializeComponent();
        }

        public bool Edit(MyExtRecordset recordset)
        {
            tbRecCount.Text = recordset.RecCount.ToString();
            bool result = ShowDialog() == System.Windows.Forms.DialogResult.OK;
            if (result)
                recordset.RecCount = int.Parse(tbRecCount.Text);
            return result;
        }
    }
}
